content/docs/use-cases/data-registries.md

```diff
@@ -120,7 +120,7 @@ $ dvc import https://github.com/example/registry images/faces
 ```
 
 Besides downloading the data, importing saves the information about the
-dependency from the local project to the data source (registry repo). This is
+dependency from the data source (registry repo) to the local project. This is
```

Comment:
Hmm no, I realize this wasn't worded clearly, but we mean the dependency that the local project has to the data source i.e. the local project depends on the data source. Let me think... ⌛

Reply:
I see what you mean, maybe "dependency of the local project on the data source (registry repo)"